File: 백준/Bronze/9501. 꿍의 우주여행/꿍의 우주여행.c
#include <stdio.h>
int main(void) {
    int t, n, d, vi, fi, ci, cnt;
    scanf("%d", &t);
    for (int i = 0; i < t; i++) {
        scanf("%d %d", &n, &d);
        cnt = 0;
        for (int j = 0; j < n; j++) {
            scanf("%d %d %d", &vi, &fi, &ci);
            if (vi * (fi / (double)ci) >= d)
                cnt += 1;
        }
        printf("%d\n", cnt);
    }
    return 0;
}

File: 백준/Bronze/9501. 꿍의 우주여행/꿍의 우주여행.py
import sys
input=lambda:sys.stdin.readline().rstrip()
for _ in range(int(input())):
    n,d=map(int,input().split())
    cnt=0
    for _ in range(n):
        vi,fi,ci=map(int,input().split())
        if vi*(fi/ci)>=d:
            cnt+=1
    print(cnt)